### ISSUE TITLE
Display Real Time Data from a RealSense Camera

### ISSUE THREAD

**Issue by tomatac** [potree/potree#1024]
Is it possible / feasible to use potree to display a point cloud in real time in the browser?

I am thinking of displaying images captured from a Intel [RealSense](https://www.intel.com/content/www/us/en/architecture-and-technology/realsense-overview.html) camera (at a rate of 1 image per second) in the browser, possibly streaming them thru [gRPC.](https://grpc.io/)
Ref: 
- [RealSense](https://github.com/IntelRealSense/librealsense)
- [gRPC](https://github.com/grpc/grpc-web)

Any suggestions would be appreciated.

**Comment by smcavoy12** [comment]
I dont think so, There's a PotreeConverter step required to make it work with the multiresolution potree pointcloud viewer, that sort of takes the "real time" part out of it.

I don't have my realsense on me to confirm this works, but I think we used this example in the past
https://intel.github.io/depth-camera-web-demo/depthdemo.html

I think it's more efficeint to translate the greyscale depth video feed as a depth material on a plane, and project the rbg feed over that.

**Comment by tomatac** [comment]
Thank you for yours suggestions @smcavoy12 . 
This example does not seem to work well with the new D400 cameras the repository does not look like is actively maintained.
(https://intel.github.io/depth-camera-web-demo/depthdemo.html)

I was thinking to display [PLC](https://pointclouds.org/) point cloud data in the browser (maybe using React). 
Do you have any suggestions for that?
